trie/snapshotTrieStorageManager_test.go

```diff
@@ -1,12 +1,15 @@
 package trie
 
 import (
+	"errors"
 	"strings"
 	"testing"
 
 	"github.com/multiversx/mx-chain-core-go/core"
 	"github.com/multiversx/mx-chain-core-go/core/check"
 	"github.com/multiversx/mx-chain-go/common"
+	errorsMx "github.com/multiversx/mx-chain-go/errors"
```

Comment:
might rename to `internalErrors`

Reply:
decided to use errorsMx in a previous PR and kept the same everywhere